src/components/XboxPage/XboxPage.jsx

```diff
@@ -0,0 +1,171 @@
+import { FilterMenu } from "../FilterMenu/FilterMenu";
+import { FilterPhone } from "../FilterPhone/FilterPhone";
+import { NavPhone } from "../NavPhone/NavPhone";
+import { Header } from "../Header/Header";
+import {
+  useXbox,
+  useCart,
+  useWishlist,
+  useNavPhone,
+  useAlert,
+} from "../../contexts";
+import { useWindowSize } from "../../hooks";
+import { AiOutlineHeart, AiFillHeart } from "react-icons/ai";
+import { IoBagCheckOutline } from "react-icons/io5";
+import "./xboxPage.css";
+
+export const XboxPage = () => {
+  const yellowColor = "rgb(255, 149, 41)";
+  const greenColor = "rgb(16, 124, 16)";
+
+  const { products, ratingFilter, priceFilter, dispatch } = useXbox();
+  const { cart, cartDispatch } = useCart();
+  const { wishlist, wishlistDispatch } = useWishlist();
+
+  const screenWidth = useWindowSize().width;
+  const { navPhoneVisible, setNavPhoneVisibility } = useNavPhone();
+
+  const { setSnackbar } = useAlert();
+
+  useEffect(() => setNavPhoneVisibility(false), [setNavPhoneVisibility]);
+
+  const filterData = (products, price, rating) => {
+    let data = [...products];
+    if (price)
+      data = data.filter(
+        (item) => item.price >= price.low && item.price < price.high
+      );
+    if (rating) data = data.filter((item) => item.rating >= rating);
+    return data;
+  };
+  const filteredData = filterData(products, priceFilter, ratingFilter);
+
+  return (
+    <div className="container-app">
+      {screenWidth < 768 && navPhoneVisible && <NavPhone active="xbox" />}
+      <Header active="xbox" />
+      <main className="main-container-products">
+        {screenWidth < 768 ? (
+          <FilterPhone
+            dispatch={(args) => dispatch(args)}
+            ratingFilter={ratingFilter}
+            priceFilter={priceFilter}
+          />
+        ) : (
+          <FilterMenu
+            dispatch={(args) => dispatch(args)}
+            ratingFilter={ratingFilter}
+            priceFilter={priceFilter}
+          />
+        )}
+        <div className="games-list">
+          <div className="product-grid">
+            {filteredData.length !== 0 &&
+              filteredData.map((item) => {
+                return (
+                  <CardCustom key={item.id}>
+                    <CardImage image={item.coverImage} title={item.name} />
+                    <CardContent>
+                      <div>
+                        <div
+                          style={{
+                            display: "flex",
+                            justifyContent: "space-between",
+                          }}
+                        >
+                          <span className="custom-container-rating">
+                            <StarRating
+                              rating={item.rating}
+                              color={yellowColor}
+                            />
+                          </span>
+
+                          {wishlist.find(

```

Comment:
The `checkWishlist`  which I mentioned above can be a function in utils as it is used across files.